bluzelled/inc/http/server/ServiceManager.h

```diff
@@ -0,0 +1,42 @@
+#ifndef KEPLER_SERVICEMANAGER_H
+#define KEPLER_SERVICEMANAGER_H
+
+#include "server.hpp"
+
+#include <boost/thread.hpp>
+
+namespace  http
+{
+    namespace  server
+    {
+        class ServiceManager
+        {
```

Comment:
@rich Brackets are in the wrong place :)

Reply:
Removed the brackets. and the namespace. I think the class bracket is in the right place...